bigquery/samples/tests/test_client_query_dry_run.py

```diff
@@ -20,6 +20,6 @@ def test_client_query_dry_run(capsys, client):
 
     query_job = client_query_dry_run.client_query_dry_run(client)
     out, err = capsys.readouterr()
-    assert "This query will process 65935918 bytes." in out
+    assert "This query will process 0 bytes." not in out
```

Comment:
Instead of using **regex**, the assert was changed to confirm that processed bytes are not equal 0.